dotnet/src/Connectors/Connectors.AI.OpenAI/AzureSdk/SKChatMessage.cs

```diff
@@ -7,11 +7,11 @@
 namespace Microsoft.SemanticKernel.Connectors.AI.OpenAI.AzureSdk;
 
 /// <summary>
-/// Chat message representation from Semantic Kernel ChatMessageBase Abstraction
+/// Chat message representation from Semantic Kernel ChatMessage Abstraction
 /// </summary>
-public class SKChatMessage : ChatMessageBase
+public class SKChatMessage : SemanticKernel.AI.ChatCompletion.ChatMessage
```

Comment:
Can we rename this as well? e.g. AzureOpenAIChatMessage?  Calling it SKChatMessage when it's actually specific to the Azure OpenAI connector, and when the base type that is specific to SK is called just ChatMessage feels wrong.

Reply:
Done